.github/workflows/macos_build_notarized.yml

```diff
@@ -182,11 +182,11 @@ jobs:
         echo "app-name=${{ env.app-name }}" >> $GITHUB_OUTPUT
         echo "dsym-name=DuckDuckGo-${{ env.app-version }}-dSYM.zip" >> $GITHUB_OUTPUT
 
-    - name: Upload app artifact
+    - name: Upload tar.xz artifact
```

Comment:
Could we stick to the old name here? It’s the app artifact anyway, just the compression method changes.